charts/values.yaml

```diff
@@ -0,0 +1,140 @@
+# Default values for growthbook.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: growthbook/growthbook
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
+
+externalSecretName: growthbook-external-secret
+
+port:
+  frontendPort: 3000
+  backendPort: 3100
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+env:
+  - name: NODE_ENV
+    value: production
+  - name: MONGODB_URI
+    valueFrom:
+      secretKeyRef:
+        name: growthbook-external-secret
+        key: mongo-connection-uri
+  - name: JWT_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: growthbook-external-secret
+        key: jwt-secret
+  - name: ENCRYPTION_KEY
+    valueFrom:
+      secretKeyRef:
+        name: growthbook-external-secret
+        key: encryption-key      
+  # Specify your app_origin, api_host
+  - name: APP_ORIGIN
+    value: https://my-app-origin.io:443
+  - name: API_HOST
+    value: https://my-api-host.io:443
```

Comment:
why 443 ? in the official documentation it's port 3000 for APP_ORIGIN and port 3100 for API_HOST